oarepo_vocabularies/ui/theme/assets/semantic-ui/js/oarepo_vocabularies_ui/form/components/VocabularySelectField/VocabularySelectField.jsx

```diff
@@ -1,48 +1,64 @@
 import React from "react";
 import { Breadcrumb } from "semantic-ui-react";
-import { I18nString, RelatedSelectField } from "@js/oarepo_ui";
+import {
+  I18nString,
+  RelatedSelectField,
+  getTitleFromMultilingualObject,
+} from "@js/oarepo_ui";
 import _join from "lodash/join";
 import PropTypes from "prop-types";
+import { search } from "../../../utils";
 
 export const serializeVocabularySuggestions = (suggestions) =>
   suggestions.map((item) => {
-    const hierarchy = item.hierarchy.ancestors_or_self;
-    const key = _join(hierarchy, ".");
-    const sections = [
-      ...hierarchy.map((id, index, { length }) => ({
-        key: id,
-        content:
-          index === 0 ? (
-            <I18nString value={item.hierarchy.title[index]} />
-          ) : (
-            <span style={{ opacity: "0.5", fontSize: "0.8rem" }}>
+    const hierarchy = item?.hierarchy?.ancestors_or_self;
+    let sections;
+    let key = item.id;
+    if (hierarchy?.length > 1) {
+      key = _join(hierarchy, ".");
+      sections = [
+        ...hierarchy.map((id, index, { length }) => ({
+          key: id,
+          content:
+            index === 0 ? (
               <I18nString value={item.hierarchy.title[index]} />
-            </span>
+            ) : (
+              <span style={{ opacity: "0.5", fontSize: "0.8rem" }}>
```

Comment:
Avoid explicit style here (we are in generic library here and noone can override this in their own repo). Rather use className and put it to breadcrumbs.overrides

Reply:
@mirekys You are right, I am generally trying to eliminate all style props from the code, but this style I just was not sure where to put, and a separate less component seemed like an overkill, but I think breadcrumb override is an OK compromise. Thanks!